examples/pycaffe/layers/pascal_multilabel_datalayers.py

```diff
@@ -0,0 +1,262 @@
+# imports
+import json, time, pickle, scipy.misc, skimage.io, caffe
+
+import numpy as np
+import os.path as osp
+
+from xml.dom import minidom
+from random import shuffle
+from threading import Thread
+from PIL import Image
+
+from pascal_multilabel_with_datalayer_tools import SimpleTransformer
```

Comment:
I don't think that I'm going to get much of a chance to have a look this weekend. If it can wait until Monday, then I don't mind doing it myself. Otherwise, feel free to edit it.